tweakwcs/matchutils.py

```diff
@@ -22,12 +22,19 @@
 
 __author__ = 'Mihai Cara'
 
-__all__ = ['MatchCatalogs', 'XYXYMatch']
+__all__ = ['MatchCatalogs', 'XYXYMatch', 'MultiMatchError']
 
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
+class MultiMatchError(RuntimeError):
+    """
+    Error indicating that multiple sources matched to a single reference
+    source.
```

Comment:
```suggestion
    Error indicating that multiple sources matched to a single reference
    source. Try different values for ``tolerance`` and ``separation``to fix this error.
```

Reply:
alternatively, maybe add a section `Raises` to `XYXYMatch.__init__` docstring to describe the two exceptions (this PR + my previous PR) and add a note suggesting corrective action to fix the error.